backend/src/controllers/controller.types.ts

```diff
@@ -1,11 +1,13 @@
-import { Request, Response, NextFunction, ParamsDictionary, Paths } from 'express';
+import {
+  Request, Response, NextFunction, ParamsDictionary, Paths,
+} from 'express';
 
 export type POST = (
-    req: Request<ParamsDictionary, {}, Paths.Hangouts.Post.RequestBody>,
-    res: Response<
+    req: Request<ParamsDictionary, {}, Paths.Hangouts.Post.RequestBody>, //eslint-disable-line
+    res: Response< //eslint-disable-line
         | Paths.Hangouts.Post.Responses.$201
         | Paths.Hangouts.Post.Responses.$400
         | Paths.Hangouts.Post.Responses.$404
     >,
-    next: NextFunction
-) => Promise<void>
\ No newline at end of file
+    next: NextFunction //eslint-disable-line
```

Comment:
I loaded up the branch and wasn't getting any linting errors when I removed the `eslint-disable-line` statements. Are they needed?

Reply:
Running lint without the `eslint-disable-line` statements gave me linting errors.

Reply:
Got it, they should probably stay in then 👍